notebooks/logging-examples/logging-experiment-failures.ipynb

```diff
@@ -0,0 +1,157 @@
+    "for _ in range(20):\n",
+    "    pipe.fit(X,y)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "852bd664",
+   "metadata": {},
+   "source": [
+    "Now, tag any experiment that doesn't have a valid `badestimator__state` parameter for failures with `exp.add_tags([\"failed\"])`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "379dc1be",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for exp in project.experiments():\n",
+    "    for p in exp.parameters():\n",
+    "        if (p.name==\"badestimator__state\" and p.value==None):\n",
+    "            exp.add_tags([\"failed\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e03bd0a",
+   "metadata": {},
+   "source": [
+    "Finally, we can now retrieve all our failed experiments by passing the `tags=[\"failed\"]` to `project.experiments()`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "1ef73ac0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Experiment(project_name='Failed Experiment', id='55f2bac0-a0c8-473b-a5f0-59219071b248', name='RubiconPipeline experiment', description=None, model_name=None, branch_name=None, commit_hash=None, training_metadata=None, tags=['failed'], created_at=datetime.datetime(2022, 5, 9, 18, 54, 12, 512110))\n",
+      "Experiment(project_name='Failed Experiment', id='cbf281bd-f110-4a46-a4e0-1bcf4ed7d383', name='RubiconPipeline experiment', description=None, model_name=None, branch_name=None, commit_hash=None, training_metadata=None, tags=['failed'], created_at=datetime.datetime(2022, 5, 9, 18, 54, 12, 516871))\n",
+      "Experiment(project_name='Failed Experiment', id='766df32f-6b1e-4f5e-b22e-868e7f5e4baf', name='RubiconPipeline experiment', description=None, model_name=None, branch_name=None, commit_hash=None, training_metadata=None, tags=['failed'], created_at=datetime.datetime(2022, 5, 9, 18, 54, 12, 525243))\n",
+      "Experiment(project_name='Failed Experiment', id='a8d2bbef-ab06-450b-b73c-acdfaaaf6e9e', name='RubiconPipeline experiment', description=None, model_name=None, branch_name=None, commit_hash=None, training_metadata=None, tags=['failed'], created_at=datetime.datetime(2022, 5, 9, 18, 54, 12, 526703))\n",
+      "Experiment(project_name='Failed Experiment', id='301c11a4-cfc9-4f5c-900e-35f55d38c539', name='RubiconPipeline experiment', description=None, model_name=None, branch_name=None, commit_hash=None, training_metadata=None, tags=['failed'], created_at=datetime.datetime(2022, 5, 9, 18, 54, 12, 539486))\n",
+      "Experiment(project_name='Failed Experiment', id='a228c707-39b8-4bf0-9813-64cf51e7036e', name='RubiconPipeline experiment', description=None, model_name=None, branch_name=None, commit_hash=None, training_metadata=None, tags=['failed'], created_at=datetime.datetime(2022, 5, 9, 18, 54, 12, 541330))\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
can we expand this cell a little to show the percentage of failed and successful experiments - basically want to show that its about 70% success/30% fail